### ISSUE TITLE
Sort companies on red flags

### ISSUE THREAD

**Issue by bkj** [gophronesis/penny-sae#54]
Right not sorting by red flag is not implemented for every type of red flag.  Need to implement for all types.  Some of these are blocked by backend data issues.

- [ ] PV Anomalies (won't do in the short term)
- [x] CROWDSAR
- [x] Symbology 
- [x] Delinquency
- [x] Trading Suspensions
- [ ] Financials (waiting on data)
- [x] OTC Associates
- [ ] Frequent change in management (possibly enhancement)

**Comment by bkj** [comment]
I am punting on a number of the "threshold" function -- returning a redflag if the result is nonzero.  This is primarily because we want to sort by number of red flags __and__ "severity" of red flags.  This involves some function that maps all of the violations to the same scale, but that's ill defined.

**Comment by bkj** [comment]
This is out of date
